src/test/java/core/basesyntax/service/FileReaderImplTest.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax.service;
+
+import core.basesyntax.service.impl.FileReaderImpl;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileReaderImplTest {
+    private static final String FILE_EXIST = "src/test/java/core/basesyntax/resources/input.csv";
+    private static final String FILE_NOT_EXIST = "directory/file.csv";
+    private static FileReader fileReaderTest;

```

Comment:
It's better to write name the of service you are testing as it is. (Without 'test' suffixes and prefixes). Fix in other places
```suggestion
    private static FileReader fileReader;

```